core/field_image.js

```diff
@@ -90,13 +110,8 @@ goog.inherits(Blockly.FieldImage, Blockly.Field);
  * @nocollapse
  */
 Blockly.FieldImage.fromJson = function(options) {
-  var src = Blockly.utils.replaceMessageReferences(options['src']);
```

Comment:
How come we no longer need to pass these through ``replaceMessageReferences``?

Reply:
Oh I removed this because I thought I added it accidentally in a previous PR, looking at the git blame that is not the case.

Readded.